packages/react-admin/src/mui/input/FileInputPreview.js

```diff
@@ -1,20 +1,14 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames';
 import IconButton from 'material-ui/IconButton';
 import RemoveCircle from 'material-ui-icons/RemoveCircle';
 import { withStyles } from 'material-ui/styles';
 
 const styles = theme => ({
-    removeButtonHovered: {
```

Comment:
I don't understand why you changed that

Reply:
Checkout the `ImageInput` component, I added JSS styles to handle mouse hover

Reply:
This is what it looks like with those modifications ![screen](https://i.imgur.com/pTVYFHM.png)